config/routes.rb

```diff
@@ -280,6 +280,7 @@
 
   # You can have the root of your site routed with "root
   get "/about" => "pages#about"
+  get "/page/crayons" => "pages#crayons"
```

Comment:
just `/crayons` instead of `/page/crayons`

Reply:
I thought it's probably better to not make it top level actually, just in case... We will eventually remove this page in favor of using Storybook, but not in this PR.

Reply:
Good call!